### ISSUE TITLE
Quesion on MaskRCNN default training script. 

### ISSUE THREAD

**Issue by XiaoSong9905** [facebookresearch/detectron2#1823]
Hi FAIR Staffs, 

I'm using d2 to do instances segmentation and run into some questions, wish you can help me. 

1. With the default MaskRCNN instance segmentation training configuration file (`mask_rcnn_R_50_FPN_3x.yaml`) + detectron2 default configuration(`detectorn2/config/default.py`),  the ResNet backbone are set to have few stage frozen (`_C.MODEL.BACKBONE.FREEZE_AT = 2`). I'm wondering why this is the default behavior? Is this because we don't want to change the first few stages' weight when using pre-train ResNet weight for it contains more general image feature information (for more general, I mean image features shared by ImageNet)? 

2. Same when using MaskRCNN instance segmentation default configuration, all batch norms are `FrozenBatchNorm2d`. But when using pre-trained weight like `"detectron2://ImageNetPretrained/MSRA/R-50.pkl"`, I felt we're supposed to set bn layers to be trainable. 

Thanks.

**Comment by ppwwyyxx** [comment]
Both are standard choices that are used in papers that detectron2 implements. 
They can be set differently but we follow the standard settings by default.

**Comment by XiaoSong9905** [comment]
Thanks!
